scripts/detect_missing_i18n.py

```diff
@@ -0,0 +1,308 @@
+    + r"|"
+    + variables
+    + r"|"
+    + urls_domains
+    + r")*\')"
+)
+
+i18n_attr_missing_regex = (
+    opening_tag_open
+    + attr_syntax
+    + r"(?:"
+    + ignore_double_quote
+    + r"|"
+    + ignore_single_quote
+    + r")[^>]*?>"
+)
+i18n_attr_warn_regex = opening_tag_open + attr_syntax + r"['\"]\$\((?!_\()"
+
+
+def terminal_underline(text: str) -> str:
+    return f"\033[4m{text}\033[0m"
+
+
+def print_analysis(
+    errtype: str,
+    filename: Path,
+    details: str,
+    spacing_base: int,
+    line_number: int = 0,
+    line_position: int = 0,
+):
+    linestr = (
+        f":{line_number}:{line_position}"
+        if line_number > 0 and line_position > 0
+        else ""
+    )
+    filestring = f'{filename}{linestr}'
+    print(
+        '\t'.join(
+            [errtype, terminal_underline(filestring).ljust(spacing_base + 12), details]
+        )
+    )
+
+
+def main(files: list[Path], skip_excluded: bool = True):
+    """
+    :param files: The html files to check for missing i18n. Leave empty to run over all html files.
+    :param skip_excluded: If --no-skip-excluded is supplied as an arg, files in the EXCLUDE_LIST slice will be processed
+    """
+
+    if not files:
+        files = [
+            Path(file_path)
+            for ddir in default_directories
+            for file_path in glob.glob(f'{ddir}**/*.html', recursive=True)
+        ]
+
+    # Figure out how much padding to put between the filename and the error output
+    longest_filename_length = max(len(str(f)) for f in files)
+    spacing_base = longest_filename_length + len(':XXX:XXX')
+
+    errcount: int = 0
+    warnings: int = 0
+
+    for file in files:
+        contents = file.read_text()
+        lines = contents.splitlines()
+
+        if skip_excluded and str(file) in EXCLUDE_LIST:
+            print_analysis(Errtype.SKIP, file, "", spacing_base)
+            continue
+
+        for line_number, line in enumerate(lines, start=1):
+
+            includes_error_element = re.search(i18n_element_missing_regex, line)
+            includes_warn_element = re.search(i18n_element_warn_regex, line)
+            includes_error_attribute = re.search(i18n_attr_missing_regex, line)
+            includes_warn_attribute = re.search(i18n_attr_warn_regex, line)
+
+            char_index = -1
+            # Element with untranslated elements
+            if includes_error_element:
+                char_index = includes_error_element.start()
+                errtype = Errtype.ERR
+                errcount += 1
+            # Element with bypassed elements
+            elif includes_warn_element:
+                char_index = includes_warn_element.start()
+                errtype = Errtype.WARN
+                warnings += 1
+            # Element with untranslated attributes
+            elif includes_error_attribute:
+                char_index = includes_error_attribute.start()
+                errtype = Errtype.ERR
+                errcount += 1
```

Comment:
Ah this is where the bug is! The errors are being counted even when we _continue_ down below. We should only +1 at the end after we run all the continues.